src/main/java/org/jabref/gui/EntryTypeView.java

```diff
@@ -168,4 +178,82 @@ private void setEntryTypeForReturnAndClose(Optional<BibEntryType> entryType) {
         viewModel.stopFetching();
         this.close();
     }
+
+    public String getDescription(BibEntryType selectedType) {
+        EntryType entryType = selectedType.getType();
+        try {
+            StandardEntryType entry = (StandardEntryType) entryType;
+            switch (entry) {
+                case Article:
+                    return Localization.lang("An article from a journal or magazine.");
```

Comment:
I think it might be reasonable to favor the biblatex documentation over bibtex's, since bibtex is a subset of biblatex and bibtex is better documented. (Or choose the documentation depending on the active mode of the currently opened library - bibtex or biblatex.)

Reply:
And please also add a comment that the description is coming from the biblatex manual, e.g. biblatex manual chapter 2

Reply:
Thank you for your feedback, i will change the description wherever I used the bibtex documentation.